db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_09_26_191433) do
+ActiveRecord::Schema[7.0].define(version: 2023_09_28_211022) do
```

Comment:
I'm not entirely sure how this happened. Hopefully it won't cause a problem for others. I did deploy to QA without a problem.

Reply:
when you have a db/migrate, it creates a new db schema with a new datestamp.  Kinda like Gemfile.lock or .rubocop_todo.yml

Reply:
I don't think this has an impact on anything so no need to worry about it ... the scema.rb is for documentation and to re-create the database without migrations if needed.